keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -281,6 +293,14 @@ def _get_unconditional_context(self):
 
         return unconditional_context
 
+    @property
+    def image_encoder(self):
```

Comment:
Maybe worth a docstring pointing to the ImageEncoder class or showing an example of its use?

Reply:
Good point - added this!